lisp/ess-julia.el

```diff
@@ -39,224 +39,14 @@
 
 (require 'compile); for compilation-* below
 (require 'ess-utils)
+(require 'julia-mode)
```

Comment:
And I just added `julia-mode.el` to `.gitignore` so it can stay there and not interfere with git.